octoprint_mrbeam/static/js/working_area.js

```diff
@@ -678,6 +678,17 @@ $(function(){
 					'mb:origin': origin
 				});
 
+				// remove hidden elements with "display:none" via a css class (svg fragment needs to be placed to use getComputedStyle())
+				let allElems = newSvg.selectAll('*[class]');
+				console.log("found elements", allElems.length);
+				for (var i = 0; i < allElems.length; i++) {
+					var el = allElems[i];
+					if (window.getComputedStyle(el.node).display === 'none') {
```

Comment:
What about visibility and all sorts of opacity or colors with alpha value of zero?

Reply:
What if display:none is set per style attribute rather than class attribute?